src/main/kotlin/vm/VM.kt

```diff
@@ -249,8 +253,8 @@ class VM(
                 O_CALL, O_VCALL, O_VCALLST -> {
                     val argCount = next().intFromV
                     val name = (if (word.opcode == O_CALL) pop() else next().value!!).asString()
-                    val subj = pop()
                     val args = buildList { repeat(argCount) { add(0, pop()) } }
+                    val subj = pop()
```

Comment:
Switched the order of stack args to O_CALL to let the optimizer morph it to VCALL/VCALLST without changes.